builder/amazon/common/template_funcs.go

```diff
@@ -36,5 +36,5 @@ func templateCleanAMIName(s string) string {
 }
 
 var TemplateFuncs = template.FuncMap{
-	"clean_ami_name": templateCleanAMIName,
+	"clean_image_name": templateCleanAMIName,
```

Comment:
here, we could just keep both funcs to avoid breaking compatibility.

Reply:
I like that; I've been thinking lately that it would be nice to have a deprecation path where we support multiple template options for one minor release cycle after changing them.

Reply:
Okay, adding deprecation warning and all !